models/custom/snowplow_media_player_user_stats.sql

```diff
@@ -0,0 +1,43 @@
+{{
+  config(
+    materialized = 'ephemeral'
+  )
+}}
+
+with prep as (
+
+  select
+    domain_userid,
+    avg(case when (video_plays + audio_plays) > 0 then avg_play_time_min end) as avg_play_time_min,
```

Comment:
Good point. Now it is a question of the usual accuracy vs table scan / db cost, because if I wanted to make this accurate I'd have to add a new cte and calculate the average using the snowplow_media_player_base table. Perhaps the solution is to rename this field to `avg_session_play_time_min` to keep the content accurate while keeping the simplicity, not sure it would bother the users that much. What do you think?